docs/reference/sailing/turning.md

```diff
@@ -1,9 +1,22 @@
 # Types of Turns
 
-In sailing, there are 4 distinct types of turns. Read the descriptions below, and observe how they fit into the diagram.
+In sailing, there are 4 distinct types of turns. Read the descriptions below, and observe how they fit into the diagrams.
 
 Note that any of these types of turn can be done in either the clockwise or counter-clockwise directions.
 
+## Classifying Types Of Turns Summary
+
+``` mermaid
+graph LR
+    B[Classify a Turn] --> C{Does the sail change<br/>sides during the turn?};
+    C --> |Yes| E{Which end of<br/>the boat is upwind<br/>during the turn?};
+    C --> |No| D{Does the<br/>boat point higher<br/>or lower at the end<br/>of the turn?};
```

Comment:
Might be good to specify higher or lower respective to what. Higher is more upwind right?

Reply:
That's correct, pointing higher is pointing closer towards straight into the wind. I was sorta hoping people would read the rest of the page to see where I defined "higher" and "lower" but its probably better that I move those definitions up here instead haha